xbmc/guilib/WindowIDs.dox

```diff
@@ -112,18 +114,18 @@ This page shows the window names, the window definition, the window ID and the s
 | FullscreenLivetvInput   | WINDOW_FULLSCREEN_LIVETV_INPUT       | 10804     | None (shortcut to fullscreenlivetv) |                                    |
 | FullscreenRadioInput    | WINDOW_FULLSCREEN_RADIO_INPUT        | 10805     | None (shortcut to fullscreenradio)  |                                    |
-| GameControllers         | WINDOW_DIALOG_GAME_CONTROLLERS       | 10820     | DialogGameControllers.xml           | @skinning_v17 **New window**<p></p> |
-| Games                   | WINDOW_GAMES                         | 10821     | MyGames.xml                         | @skinning_v18 **New window**<p></p> |
-| GameOSD                 | WINDOW_DIALOG_GAME_OSD               | 10822     | GameOSD.xml                         | @skinning_v18 **New window**<p></p> |
-| GameVideoFilter         | WINDOW_DIALOG_GAME_VIDEO_FILTER      | 10823     | DialogSelect.xml                    | @skinning_v18 **New window**<p></p> |
-| GameStretchMode         | WINDOW_DIALOG_GAME_STRETCH_MODE      | 10824     | DialogSelect.xml                    | @skinning_v18 **New window**<p></p> |
-| GameVolume              | WINDOW_DIALOG_GAME_VOLUME            | 10825     | DialogSlider.xml                    | @skinning_v18 **New window** See https://github.com/xbmc/xbmc/pull/12765<p></p> |
-| GameAdvancedSettings    | WINDOW_DIALOG_GAME_ADVANCED_SETTINGS | 10826     | DialogAddonSettings.xml             | @skinning_v18 **New window**<p></p> |
-| GameVideoRotation       | WINDOW_DIALOG_GAME_VIDEO_ROTATION    | 10827     | DialogSelect.xml                    | @skinning_v18 **New window**<p></p> |
-| GamePorts               | WINDOW_DIALOG_GAME_PORTS             | 10828     | DialogGameControllers.xml           | @skinning_v20 **New window** See https://github.com/xbmc/xbmc/pull/20505<p></p> |
-| InGameSaves             | WINDOW_DIALOG_IN_GAME_SAVES          | 10829     | DialogSelect.xml                    | @skinning_v20 **New window** Part of the GSoC 2020 Saved Game Manager<p></p> |
-| GameSaves               | WINDOW_DIALOG_GAME_SAVES             | 10830     | DialogSelect.xml                    | @skinning_v20 **New window** Part of the GSoC 2020 Saved Game Manager<p></p> |
-| GameAgents              | WINDOW_DIALOG_GAME_AGENTS            | 10831     | DialogGameControllers.xml           | @skinning_v21 **New window** See https://github.com/xbmc/xbmc/pull/23548<p></p> |
+| GameControllers         | WINDOW_DIALOG_GAME_CONTROLLERS       | 10820     | DialogGameControllers.xml           | **New window** |
+| Games                   | WINDOW_GAMES                         | 10821     | MyGames.xml                         | **New window** |
+| GameOSD                 | WINDOW_DIALOG_GAME_OSD               | 10822     | GameOSD.xml                         | **New window** |
+| GameVideoFilter         | WINDOW_DIALOG_GAME_VIDEO_FILTER      | 10823     | DialogSelect.xml                    | **New window** |
```

Comment:
if you remove all `@skinning_vXY` tags you lose the purpose of providing these info, since IMO its more important to have a track on which kodi release the changes are made

Reply:
Do you have a suggestion because I couldn't find a solution online?

Reply:
What version of doxygen are you using?  I do not have this formatting issue using v1.9.8 (Ubuntu 24.04.01).

Reply:
> What version of doxygen are you using?

Version: 1.13.2

Reply:
> i have just runned generation locally also with doxygen 1.9.4 looks ok

What about when using https://xbmc.github.io/docs.kodi.tv/master/kodi-base/d2/dd9/window_ids.html?

In both Brave and Edge it's broken for me:

![image](https://github.com/user-attachments/assets/f606dbcf-70c7-424e-b497-2f255fbb749c)

Reply:
until version 1.11.0 works good, the problem is started from 1.12.0
maybe on the repository we can find some info